### PR TITLE
Add github pages documentation website

### PR DESCRIPTION
**Pull request by SijmenHuizenga**
https://fs-online.github.io/Driverless-Competition-Simulator/

### CODE REVIEW

File: ros/src/fsds_ros_bridge/src/airsim_ros_wrapper.cpp [hunk 1]
```diff
@@ -83,7 +83,7 @@ void AirsimROSWrapper::initialize_ros()
     double update_gps_every_n_sec;
     double update_imu_every_n_sec;
     double publish_static_tf_every_n_sec;
-    nh_private_.getParam("is_vulkan", is_vulkan_);
+    nh_.getParam("ros_bridge/is_vulkan", is_vulkan_);
```
Comment (davidoort) [inline ros/src/fsds_ros_bridge/src/airsim_ros_wrapper.cpp:86]: is this by mistake? it doesn't break anything but still

**Comment by davidoort** [comment]
image
There might be more of these. I think this is intended to be bullet-pointed

File: ros/src/fsds_ros_bridge/src/airsim_ros_wrapper.cpp (hunk 1 above)
Comment (SijmenHuizenga) [reply ros/src/fsds_ros_bridge/src/airsim_ros_wrapper.cpp:86]: wohops wil revert this

**Comment by SijmenHuizenga** [comment]
@davidoort thanks for pointing this out! I solved all formatting problems that I could find.

**Pull request merged**